src/app/api/dataset.api.ts

```diff
@@ -27,7 +27,7 @@ export class DatasetApi {
         private datasetLineageGQL: GetDatasetLineageGQL,
     ) {}
 
-    public getDatasetOverview(params: {
+    public getDatasetMainData(params: {
```

Comment:
Since we've renamed the function, let's be consistent and rename the GraphQL parts too.